### ISSUE TITLE
Firefox is still trying to download updates

### ISSUE THREAD

**Issue by glasserc** [NixOS/nixpkgs#56098]
## Issue description

Firefox Nightly (installed by https://github.com/mozilla/nixpkgs-mozilla/) pops up periodic messages saying that it can't download updates and prompting the user to install a new version.

In https://github.com/NixOS/nixpkgs/pull/47330, @garbas tried to patch the default preferences to disable Firefox from updating itself automatically. However, as of https://bugzilla.mozilla.org/show_bug.cgi?id=1420514, these preferences are not the intended way to do this any more; instead, we should be making this kind of change using the "policy engine". I don't know exactly how to do that.

### Steps to reproduce

- Install Firefox Nightly
- Use it for a few hours
- Observe the pop-up

## Technical details

 - system: `"x86_64-linux"`
 - host os: `Linux 4.14.98, NixOS, 18.09.2203.9bd45dddf81 (Jellyfish)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.1.3`
 - channels(ethan): `""`
 - channels(root): `"nixos-18.09.2203.9bd45dddf81, nixos-stable-18.03.132768.94d80eb7247"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

**Comment by callahad** [comment]
The policy engine is discussed here: https://github.com/mozilla/policy-templates/blob/master/README.md

TL;DR: We need to add a file, `distribution/policies.json`, in the same directory with the unpacked firefox binary (e.g., the same folder that holds `omni.ja`) with the contents:

```json
{
  "policies": {
    "DisableAppUpdate": true
  }
}
```

**Comment by callahad** [comment]
Looks like this was already done for `nixpkgs.firefox-bin` in ba2fe3c9a626a8fb845c786383b8b23ad8355951

**Comment by glasserc** [comment]
I haven't experienced this in a while, so closing.
